python_pellet/agent.py

```diff
@@ -121,12 +126,16 @@ def eelearn(self, replay_memory):
             self.EEnet.backpropagate(self.EEnet(torch.cat(merged).to(device=self.device)), targ_mix)
 
     def find_current_partition(self, state, partition_memory):
+        min_distance = np.inf
```

Comment:
```suggestion
        min_distance = np.Inf
```
`Inf` og ikke `inf` right?